app/src/components/fields/dynamicFieldsSection/utils.js

```diff
@@ -42,17 +45,23 @@ export const normalizeFieldsWithOptions = (fields, defaultOptionValueKey = VALUE
     if (!field?.definedValues?.length) {
       return field;
     }
+    if (!field.required && field.fieldType === OPTION_TYPE) {
```

Comment:
Let's not rely on a particular `fieldType` here as it can vary from one BTS to another.
Let's set it for all not required fields with `definedValues` as agreed with backend
![image](https://github.com/user-attachments/assets/347ed6ff-dcd1-40c9-a0fb-8c299e8fc19a)